test/runtests.jl

```diff
@@ -76,6 +76,7 @@ include("typetree.jl")
         # XXX invalidation does not work on Julia 1.8
         include("ruleinvalidation.jl")
     end
+    include("rules/LinearAlgebra/blas.jl")
```

Comment:
Can we have this run, even if enzymerules are not supported (thus to test tablegen and fallback blas)?

Reply:
similarly can we move the test file into just tests rather than rules?